jrnl/jrnl.py

```diff
@@ -316,17 +317,36 @@ def _delete_search_results(journal, old_entries, **kwargs):
 
 
 def _display_search_results(args, journal, **kwargs):
-    if args.short:
+    if args.short or args.export == "short":
         print(journal.pprint(short=True))
 
+    elif args.export == "pretty":
+        print(journal.pprint())
+
     elif args.tags:
         print(plugins.get_exporter("tags").export(journal))
 
     elif args.export:
-        exporter = plugins.get_exporter(args.export)
-        print(exporter.export(journal, args.filename))
+        _export_journal(args, journal)
```

Comment:
Let's get rid of _export_journal to keep things simple:

```suggestion
        exporter = plugins.get_exporter(args.export)
        print(exporter.export(journal, args.filename))
```